### ISSUE TITLE
Light not working -8 and below

### ISSUE THREAD

**Issue by Dip-Set1** [opentibiabr/otclient#60]
Tl;dr - Light doesn't work -8 and below. Being honest, I included some changes to client sources, but none involving light system. I am not 100% if this is problem with fresh Mehah build. Without going into exhaustive detail, in order to test locally on my server, I must make some changes to force server port, ip, client version protocols. I did NOT touch light system in any way, so I believe this may be a valid issue with Mehah OTClient fresh compile. 
https://media.discordapp.net/attachments/731630562267234475/815682223365816360/unknown.png
### Before creating an issue, please ensure:
- [X] This is a bug in the software that resides in this repository, and not a
      support matter (use https://otland.net/forums/otclient.494/ for support)
- [O] This issue is reproducible without changes to the code in this repository

### Steps to reproduce (include any configuration/script required to reproduce)
1. Compile client without change to light system, only added encryption/cam system
2. Go to floor -8
3. There is no light source visible.

### Expected behaviour
Light should function at floor -8 and below

### Actual behaviour
Light only functions on ground floor and above

### Environment
7.72 OTClient latest build as of 02/28/2021
Compiled with Windows 10 Home, Visual Studio 2019
